cytotable/utils.py

```diff
@@ -107,13 +108,13 @@ def _parsl_loaded() -> bool:
     try:
         # try to reference Parsl dataflowkernel
         parsl.dfk()
-    except RuntimeError as rte:
-        # if we detect a runtime error that states we need to load config
+    except ConfigurationError as pce:
+        # if we detect a Parsl ConfigurationError that states we need to load config
         # return false to indicate parsl config has not yet been loaded.
-        if str(rte) == "Must first load config":
+        if pce.args[0] == "Must first load config":
```

Comment:
I looked at the Parsl source and, sure enough, there are a number of places where `ConfigurationError` is raised that are differentiated just by the exception's message text. While I also don't love checking the message text, since the Parsl maintainers could decide to tweak the text at any time and this would break, I don't see any way around it currently. Perhaps it's something to raise with the Parsl devs?

Reply:
Great points, thanks! I reached out to Parsl devs via Parsl/parsl#2873 for a [suggested approach](https://github.com/Parsl/parsl/issues/2873#issuecomment-1726540856). I also updated the PR description to reference and not close #94 (as this would suggest we need a possible further update).